packages/fiori/src/themes/NotificationListItem.css

```diff
@@ -143,3 +143,8 @@
 [dir="rtl"] .ui5-nli-footer-showMore {
 	margin-right: 1rem;
 }
+
+[dir=rtl] .ui5-prio-icon {
+	padding-left: .5rem;
+	padding-right: 0rem;
+}
```

Comment:
This code will fit better in the file `NotificationPrioIcon.css`.

Reply:
Done